### ISSUE TITLE
ReactDOM findDOMNode is deprecated

### ISSUE THREAD

**Issue by wereHamster** [JohannesKlauss/react-hotkeys-hook#314]
https://reactjs.org/docs/react-dom.html#finddomnode

The `findDOMNode()` function call (and dependency on `react-dom`) has been added in 34774f9f07cb3ae12bfb358a391aa0a2df30fb79.

AFAIU the code, `findDOMNode()` shouldn't be needed, you can compare the current ref with the active element directly, since the type of `ref.current` extends `Element`:

```
  if (ref.current === null || document.activeElement === ref.current) {
```

**Comment by JohannesKlauss** [comment]
Good catch, should be just fine. I'll release a fix in a few minutes.

**Comment by JohannesKlauss** [comment]
Fixed in `2.2.1`
